.github/workflows/bank-api-library.check.yml

```diff
@@ -145,6 +145,13 @@ jobs:
 
       - name: run android lint
         run: ./gradlew bank-api-library:library:lint
+        with:
+          script: >
+            -PtestClientId="gini-mobile-test"
+            -PtestClientSecret="${{ secrets.GINI_MOBILE_TEST_CLIENT_SECRET }}"
```

Comment:
@a-szotyori 
Without this parameters lint failed with new gradle plugin.
Is it ok to add this parameters like this?

Reply:
Interesting why the lint gradle task runs "androidTest" tasks which we configured to depend on our "injectTestProperties" task 🤔 

Could you check why the linter runs "androidTest" tasks? If you can't find a way to make the linter independent from "androidTest", then we can keep your fix.

Reply:
I can't find how to make linter independent. I think we need to create a task for this, and return to it later.

Reply:
Thank you for checking it! Then we can keep your fix. 👍

Reply:
@a-szotyori 
Ok, can you approve the PR?